kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/SchemaRegistryService.java

```diff
@@ -61,7 +64,8 @@
   public Mono<String[]> getAllSubjectNames(String clusterName) {
     return clustersStorage.getClusterByName(clusterName)
         .map(cluster -> webClient.get()
-            .uri(cluster.getSchemaRegistry() + URL_SUBJECTS)
+            .uri(cluster.getSchemaRegistry().getAddress() + URL_SUBJECTS)
+            .headers(headers -> setBasicAuthIfEnabled(cluster, headers))
```

Comment:
could we set it on webclient level?

Reply:
Here we can make requests for different clusters and in each cluster can be it's own Schema Registry with it's own settings. So we can't use pre configured web client

Reply:
I added method which returns configured WebClient